src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Shape implements Drawable {
```

Comment:
you should implement an interface with getArea method, and rename Shape to Figure
```suggestion
public abstract class Shape implements Drawable {
```